Dockerfile

```diff
@@ -17,7 +17,11 @@ RUN make juno
 RUN upx /app/build/juno
 
 # Stage 2: Build Docker image
-FROM alpine:3.14 AS runtime
+FROM alpine:3.18 AS runtime
```

Comment:
`build` is container for building result binaries and `runtime` is container that contains only what necessary to run result binary (because we don't need all of these `build-base clang upx cargo go pkgconfig` to run juno container). This way result docker image gets smaller